apps/cowswap-frontend/src/modules/account/containers/AccountDetails/index.tsx

```diff
@@ -80,9 +81,30 @@ export function renderActivities(activities: ActivityDescriptors[]) {
   )
 }
 
-export function getStatusIcon(connector: Connector, walletDetails?: WalletDetails, size?: number) {
+export function getStatusIcon(
```

Comment:
Would be better to convert the function to a react component and have image loading state inside

Reply:
And extract it to a separate file of course

Reply:
It looks perfect now, thanks!